docs/asyncio_api.rst

```diff
@@ -4,47 +4,44 @@
 asyncio API
 ===========
 
-Create an instance of the ``AsyncClient`` to use the asyncio client API.
+Create an instance of the ``AsyncZyteAPI`` to use the asyncio client API.
 You can use the method ``request_raw`` to perform individual requests:
 
 .. code-block:: python
 
     import asyncio
-    from zyte_api.aio.client import AsyncClient
-
-    client = AsyncClient(api_key="YOUR_API_KEY")
+    from zyte_api import AsyncZyteAPI
 
+    client = AsyncZyteAPI(api_key="YOUR_API_KEY")
 
     async def single_request(url):
-        return await client.request_raw({"url": url, "browserHtml": True})
-
+        return await client.get({"url": url, "browserHtml": True})
 
     response = asyncio.run(single_request("https://books.toscrape.com"))
-    # Do something with the response…
 
 .. tip:: You can skip the ``api_key`` parameter if you :ref:`use an environment
     variable instead <api-key>`.
 
-There is also ``request_parallel_as_completed`` method, which allows
-to process many URLs in parallel, using multiple connections:
+There is also an ``iter`` method, which allows to process many URLs in
+parallel, using multiple connections:
 
 .. code-block:: python
 
     import asyncio
     import json
     import sys
 
-    from zyte_api.aio.client import AsyncClient, create_session
+    from zyte_api import AsyncZyteAPI, create_session
     from zyte_api.aio.errors import RequestError
 
     async def extract_from(urls, n_conn):
-        client = AsyncClient(n_conn=n_conn)
+        client = AsyncZyteAPI(n_conn=n_conn)
         requests = [
             {"url": url, "browserHtml": True}
             for url in urls
         ]
         async with create_session(n_conn) as session:
-            res_iter = client.request_parallel_as_completed(requests, session=session)
+            res_iter = client.iter(requests, session=session)
             for fut in res_iter:
```

Comment:
Does the order in res_iter match the order or items in `requests`? Should we document it?

Reply:
eafd59f